### PR TITLE
implemented the car card and main page grid.

### PR DESCRIPTION
**Pull request by moath-dawood**

### CODE REVIEW

File: src/Components/CarsMain/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import PopularCar from '../CarGrid'
+import { Button, Grid, Typography } from '@mui/material'
+export const CarsMain = () => {
+    const PopularCars = [
```
Comment (baha-ott) [inline src/Components/CarsMain/index.js:5]: move data to a seperate file

File: src/Components/CarGrid/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Grid, Typography, Button } from '@mui/material'
+import CarCard from '../CarCard'
+import React from 'react'
+
+const index = ({ name, button, cars }) => {
```
Comment (baha-ott) [inline src/Components/CarGrid/index.js]: do not use index as a name of the Component!

File: src/Components/CarCard/index.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
+import { Box, Grid, Typography, Button, Link } from '@mui/material'
+import FavoriteBorderIcon from '@mui/icons-material/FavoriteBorder';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import IconButton from '@mui/material/IconButton';
+import LocalGasStationIcon from '@mui/icons-material/LocalGasStation';
+import GroupIcon from '@mui/icons-material/Group';
+import DonutLargeIcon from '@mui/icons-material/DonutLarge';
+import React from 'react'
```
Comment (baha-ott) [inline src/Components/CarCard/index.js]: should be at the top of the file

File: src/Components/Homepage/index.js [hunk 4]
```diff
@@ -1,9 +1,10 @@
 import React from 'react'
+import CarsMain from '../CarsMain'
 
 export const HomePage = () => {
   return (
     <div>
```
Comment (baha-ott) [inline src/Components/Homepage/index.js:7]: use main element instead of just a regular div , and every section of the website should be in section element

Comment (moath-dawood) [reply src/Components/Homepage/index.js:7]: Will change it after I merge the pull requests and add the components into the HomePage.

File: src/Components/CarsMain/index.js (hunk 1 above)
Comment (moath-dawood) [reply src/Components/CarsMain/index.js:5]: this is only a mock data to test the functionality of the car card.

**Comment by moath-dawood** [comment]
All the requested changes have been implemented.

**Pull request merged**